src/textual/css/_styles_builder.py

```diff
@@ -119,11 +130,46 @@ def process_visibility(
                     self.error(
                         name,
                         token,
-                        f"invalid value for visibility (received {value!r}, expected {friendly_list(VALID_VISIBILITY)})",
+                        f"property 'visibility' has invalid value {value!r}; expected {friendly_list(VALID_VISIBILITY)}",
                     )
             else:
                 self.error(name, token, f"invalid token {value!r} in this context")
 
+    def process_opacity(self, name: str, tokens: list[Token], important: bool) -> None:
```

Comment:
Will need to have a think about how the `process_*` methods can be covered by tests. We perhaps don't want to test the StyleBuilder directly, and instead ensure we have solid testing around the parsing functions it's used within.